### ISSUE TITLE
recoConversion comparison instability in WF 23234.0

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/cmssw#32338]
Spurious differences in recoConversion quantities have started appearing in WF 23234.0 (2026 D49):
![all_OldVSNew_TTbar14TeV2026D49wf23234p0c_recoConversions_allConversions__RECO_obj_nTracks](https://user-images.githubusercontent.com/4672808/100632014-f393f800-32f1-11eb-94c0-36611d6d1ea1.png)

Observed in the following PR tests so far:
#32324
#32326
#32327

**Comment by kpedro88** [comment]
assign reconstruction,upgrade

**Comment by guitargeek** [comment]
The PR tests you quote use CMSSW_11_3_X_2020-11-29-0000, right?

Would that mean that the effects of all PRs merged since then would show up in the comparisons

I'm just worried that this issue might be related to my PR https://github.com/cms-sw/cmssw/pull/32266 that indeed touched the conversion producer code, and this PR was one of two that was merged since the release that was used for the comparison baseline I think:

https://cmssdt.cern.ch/SDT/html/showIB.html

However, https://github.com/cms-sw/cmssw/pull/32266 only moved code from some `.h` to `.cc` files and occasionally did the `getByToken()` to `getHandle()` modernization. So I don't know if it's actually related.

**Comment by kpedro88** [comment]
Usually, the bot posts a notification if it merges additional PRs on top of the IB. I don't see that in this case.

**Comment by perrotta** [comment]
> The PR tests you quote use CMSSW_11_3_X_2020-11-29-0000, right?
> 
> Would that mean that the effects of all PRs merged since then would show up in the comparisons
>

No, only the PRs merged before CMSSW_11_3_X_2020-11-29-0000 can be responsible. Your own was only merged later on in CMSSW_11_3_X_2020-11-29-2300

**Comment by slava77** [comment]
is there any indication of the problem before CMSSW_11_3_X_2020-11-29-0000? 
I'm thinking that the point was the switch to 11_3_X (new gcc version and what not) and that everything after we switched until 11-29-0000 was just not visible as a problem due to the issues with IBs and the tests.

**Comment by kpedro88** [comment]
I hadn't noticed the problem before today (but was offline for the holiday the past few days). The gcc version change is worth checking.

**Comment by smuzaffar** [comment]
@kpedro88 , is this still an open issue?

**Comment by kpedro88** [comment]
@smuzaffar can you fix the bot so it stops removing my labels?

I don't recall observing these spurious differences recently, but I'm also not aware of a particular PR that would have fixed them. Current L2s are probably more attentive to this than I am.

**Comment by jpata** [comment]
This happened again in https://github.com/cms-sw/cmssw/pull/36910 for 12_1_X, ZMM13TeV2017wf10042p0.
